WordPress/build.gradle

```diff
@@ -331,6 +339,22 @@ dependencies {
     implementation (project(path:':libs:editor:WordPressEditor')) {
         exclude group: 'org.wordpress', module: 'utils' 
     }
+
+    // Hermes is a React Native dependency that we somehow lost at runtime
+    // since moving to the Bintray binary for Gutenberg.
+    //
+    // These cannot be defined in the WordPressEditor project itself, or we'll
+    // get a "Direct local .aar file dependencies are not supported when
+    // building an AAR." error.
+    //
+    // FIXME: In the long run, we'll want to read this from the Gutenberg AAR.
+    //
+    // A starting point might be the fact that the
+    // @wordpress_react-native-bridge-release AAR contains only the
+    // -release.aar version of Hermes in its libs folder
+    debugRuntimeOnly files('./../libs/hermes/hermes-debug.aar')
+    releaseRuntimeOnly files('./../libs/hermes/hermes-release.aar')
+
```

Comment:
I think this is supposed to be a part of the `WordPressEditor` module as well. WordPress module shouldn't need the hermes binary.

I know this is just a temporary hack 🤞 but I think it'd be good to keep things in their proper place so as to not confuse ourselves.

Reply:
>  I think it'd be good to keep things in their proper place so as to not confuse ourselves.

💯 thank you for pointing this out. I'll have a look.

Reply:
Hey @oguzkocer, I had forgotten about my comment regarding the "Direct local .aar file dependencies are not supported when building an AAR" error" and I tried moving the these statements in the `build.gradle` for the editor. I got that error again.

As far as I can understand, we can't have an AAR that includes another AAR.

This is the full message:

> Direct local .aar file dependencies are not supported when building an AAR. The resulting AAR would be broken because the classes and Android resources from any local .aar file dependencies would not be packaged in the resulting AAR. Previous versions of the Android Gradle Plugin produce broken AARs in this case too (despite not throwing this error).

This SO answer [suggests](https://stackoverflow.com/a/63665094/809944) to import the second AAR (in our case the Hermes one) from a repository, but I think we should be able to find a way to make the `gutenberg` code package it for us properly (see https://github.com/orgs/wordpress-mobile/projects/80#card-52385821).

I could address part of the "keep things in their proper place so as to not confuse ourselves" by moving the lib in the editor subfolder, but the includes would still be here. What do you think? Or have you got another suggestion on how to get this to work? 🙏

Reply:
[This](https://github.com/wordpress-mobile/WordPress-Android/pull/13014/files#diff-46e543167c983e22ee4eed026dfc1ad6624f9878ef3b7826acea65b402811b97R97-R99) is the way we have done it before, as a temporary measure of course. Does that answer your question?

Reply:
Thank you @oguzkocer. I did try that approach originally and it resulted in that "Direct local .aar file dependencies are not supported when building an AAR..." error.

I updated it to what's in f2cf853 and it works when running from `./gradlew` and from Android Studio both.

Hopefully it won't be a long lived measure anyway, but I'm glad you helped me make it better.